### PR TITLE
Login/Signup done

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**

### CODE REVIEW

File: bookclub/settings.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 # Application definition
```
Comment (asamolion) [inline bookclub/settings.py]: add a requirements.txt file. Google what that is.

File: books/templates/books/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+</form>
+
+<br/>
+
+<p>New here? Create a free account!</p>
+
+<form action="{% url 'books:signup' %}" method="post">
+    {% csrf_token %}
+    <input type="text" name="username" placeholder="Username"> <br/>
+    <input type="password" name="password" placeholder="••••••••"> <br/>
```
[the remark this answers is not in the document]
Comment (Ali-D-Akbar) [reply books/templates/books/index.html]: done

File: bookclub/settings.py (hunk 1 above)
Comment (Ali-D-Akbar) [reply bookclub/settings.py]: done

Comment (asamolion) [reply bookclub/settings.py]: Can you please tell me how you generated this requirements file?

Comment (Ali-D-Akbar) [reply bookclub/settings.py]: pip freeze > development.txt

Reference: https://www.patricksoftwareblog.com/setting-up-the-django-project-part-vi-environments-for-development-vs-production/

Comment (asamolion) [reply bookclub/settings.py]: image


I'm getting this error. Resolve it please. 

Take the help of one of the UCSD team if you need.

**Comment by Ali-D-Akbar** [comment]
Use python3 instead of python:

python3 manage.py runserver

File: bookclub/settings.py (hunk 1 above)
Comment (Ali-D-Akbar) [reply bookclub/settings.py]: Use python3 instead of python:

python3 manage.py runserver

**Pull request merged**